client/src/components/Button/index.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
```

Comment:
Either you put a semicolon or you don't
follow one pattern and use it in all files 
I prefer you don't put a  semicolon in react

Reply:
Done